src/listpack.c

```diff
@@ -504,22 +563,28 @@ unsigned char *lpFirst(unsigned char *lp) {
 
 /* Return a pointer to the last element of the listpack, or NULL if the
  * listpack has no elements. */
+/* 返回一个指向 listpack 最后一个元素的指针，如果 listpack 没有元素则返回 NULL。*/
 unsigned char *lpLast(unsigned char *lp) {
-    unsigned char *p = lp+lpGetTotalBytes(lp)-1; /* Seek EOF element. */
-    return lpPrev(lp,p); /* Will return NULL if EOF is the only element. */
+    unsigned char *p = lp+lpGetTotalBytes(lp)-1; /* Seek EOF element. */ /* 找到 EOF 元素。*/
+    return lpPrev(lp,p); /* Will return NULL if EOF is the only element. */ /* 如果只有 EOF 元素则返回NULL。*/
```

Comment:
```suggestion
    return lpPrev(lp,p); /* Will return NULL if EOF is the only element. */ /* 如果只有 EOF 元素则返回 NULL。*/
```
这里可以起一个新行写中文注释，和英文注释写在同一行以后合并新版本的时候会冲突